### DISCUSSION TITLE
Example for Vite + React + Tamagui (web) ?

### DISCUSSION THREAD

**Discussion by thevipinmishra** [tamagui/tamagui#466] [Q&A]
Hi there, I'm pretty excited for the v1 to try out.

I'm going ahead with Vite integration, following the docs, I'm having issues with the configuration.

First of all, links which are in Vite docs are 404. ([https://github.com/tamagui/tamagui/blob/master/apps/site-vite](https://github.com/tamagui/tamagui/blob/master/apps/site-vite), [https://tamagui.dev/docs/guides/Installation](https://tamagui.dev/docs/guides/Installation), [https://tamagui.dev/docs/guides/Configuration](https://tamagui.dev/docs/guides/Configuration).

Also, I'm trying the Snippet from the docs page, and it's kind of not working for me.

Here's my `vite.config.js`

```js
import { defineConfig } from "vite";
import react from "@vitejs/plugin-react";
import { tamaguiExtractPlugin, tamaguiPlugin } from "@tamagui/vite-plugin";

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [
    react(),
    tamaguiPlugin({
      components: ["tamagui"],
    }),
    tamaguiExtractPlugin(tamaguiConfig),
  ],
});
```

After `yarn dev`, I'm getting this error in my terminal.

![image](https://user-images.githubusercontent.com/52159805/210032840-72d41e21-c0a5-4c83-ba0f-c12c4ce44aa3.png)

Can we have a pre-made example for Vite please?

**Comment by bureyburey** [comment]
@thevipinmishra try making the following changes:

```
import vitePlugins from "@tamagui/vite-plugin";

const { tamaguiExtractPlugin, tamaguiPlugin } = vitePlugins;
```

**Comment by bureyburey** [comment]
@natew 
getting a new error after upgrading to latest (1.0.17)

might be related to this [change](https://github.com/tamagui/tamagui/commit/7ca47c08555dc1830f93af634820bad880d4afb4)?

![image](https://user-images.githubusercontent.com/20264976/213432172-1127c2a0-0178-462a-b878-414478d52289.png)
